### PR TITLE
Adds New Varadero with minimal fishing code

### PR DESCRIPTION
**Pull request by Agameofscones**
## About The Pull Request

New map, fishing as a minimal system for building off of (no solid content yet, leaving it partially drafted).

newVaradero_withComms

Fishing code by Geeves
Pred ship detailing and layout by Spartanbobby,
Gullible Toothfish and beer coolers by Monkeysfist101
Original map by Infernus.

## Why It's Good For The Game

New map, if I don't PR this now, I never will because fishing is way to large of a project even for a non-gameplay mechanic.

## Changelog

:cl: Triiodine, Geeves, Spartanbobby, Monkeysfist101
add: Added New Varadero, a re-detailed, but otherwise almost entirely unchanged Ice Colony LZ2 underground with some modern amenities as a standalone map.
experiment: Lowered the population minimum to 100 from 130 for the following: Science Annex, CORSAT, Trijent Dam, Sorokyne Strata, and Chance's Claim (we're not hitting the 150 avg anymore guys)
experiment: Ice Colony (v2 legacy) and CORSAT are now disabled and cannot be forced by admins, pending maintenance.
code: Added fishing backend by Geeves. Not in normal gameplay, admin spawn only.
/:cl:

### CODE REVIEW

**Comment by AnCopper** [comment]
Does this place only have one LZ?

**Comment by Unknownity** [comment]
all the resin nodes need to be replaced with the setup/distress/xeno_weed_node spawners instead so the weeds don't spawn in extended for example

**Comment by morrowwolf** [comment]
The areas around the LZ should also have the initial weed protection. It's a bit too close as it stands.

**Comment by morrowwolf** [comment]
image

File: maps/new_varadero.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "monkey_types": [
+        "farwa"
+    ],
+    "traits": [{ "Ground": true }],
+    "xvx_hives": {
+        "xeno_hive_alpha": 0,
+        "xeno_hive_bravo": 0,
+        "xeno_hive_charlie": 0,
+        "xeno_hive_delta": 0
+    }
```
Comment (Segrain) [inline maps/new_varadero.json:35]:
```suggestion
    },
    "gamemodes": [
        "Distress Signal",
        "Hunter Games",
        "Hive Wars",
        "Faction Clash",
        "Infection"
    ]
```
image

**Review by nauticall — CHANGES_REQUESTED** [review]
image
image

Inconsistent door floorings in security (blue to red, and green to yellow). Most doors have plating under them in this map so maybe put plating under the rest of the doors as well. Or do what other SS13 maps do and put normal flooring under them all instead, but that needs a bit more change.

image
Yellow: These are normal reinforced walls compared to the indestructible ones that line the LZ. Xenos can break through these. Don't know if intentional or not.
Orange: These are also easily breakable and give xenos a point of entry into the LZ via the water. Again, no idea if intentional or not.

image
Maybe furnish these rooms? They look a bit tacked-on without any furniture inside of them.

image
Orange: Add a platform here for height consistency.
Yellow: Maybe put some sort of incinerator or recycler (like on the Almayer) connected to a conveyor here, SS13-style? Looks like garbage here would just pile up otherwise.

image
Pretty, but admittedly a bit funky to grow dirty organic plants in the same room as sensitive electronic equipment. Maybe make the area a bit more sterile.

image
Earmuffs in this area, maybe, for Responsible Shooting:tm:?

Think that's it (for now >:) ). Great map! Love the detail.

**Comment by Wintermote** [comment]
Played once so far as IO, the compactness is very nice. Felt like there was a LOT of intel, but that may simply be due to the compact nature of the map. It definitely felt good pushing up and being able to grab a crapton of intel, but at the same time the compactness makes it hard to grab intel by going alone anywhere, so it probably balances itself out overall.

It's nice to see CM getting more maps

**Review by Nanu308 — CHANGES_REQUESTED** [review]

**Review by Nanu308 — APPROVED** [review]
lgtm map wise

**Comment by Nanu308** [comment]
Sprites approved by @monkeysfist101

**Pull request merged**